app/components/candidate_interface/application_dashboard_guidance_component.html.erb

```diff
@@ -10,7 +10,7 @@
 
 <% elsif has_multiple_offers_but_awaiting_decisions? %>
 
-  <p class="govuk-body"><%= number_of_offers_in_words %> of your training providers have made a decision on your application.</p>
+  <p class="govuk-body">2 of your training providers have made a decision on your application.</p>
```

Comment:
Could this number not also be 3 (in which case it should possibly say ‘All’). Or is there a separate state for that?

Reply:
You can't be awaiting a decision and have 3 offers right? We will have to revisit this once a candidate can apply to 5 choices though.

Reply:
Wouldn't it just make sense to use `<%= number_of_offers %>` so that we don't have to revisit?

Reply:
Yea tbh i didn't really think about that at the time. I'll revert now

Reply:
So i think it would end up being something like

`  <p class="govuk-body"><%= @application_form.application_choices.offer.count %> of your training providers have made a decision on your application.</p> 
`

right? Since it should be numbers rather than words.